src/KeymapsDialog.ts

```diff
@@ -0,0 +1,236 @@
+import { html, render } from 'lit-html'
+import { PluginOptions } from 'grapesjs'
+import { KeymapsDialogManager, titleCase } from './KeymapsDialogManager'
+
+const defaultCSS = `
```

Comment:
Later, it might be a good idea to retrieve the CSS from an external file and use SCSS.

Reply:
+1